packages/common/src/store/account/sagas.ts

```diff
@@ -54,7 +73,90 @@ function* handleFetchTrackCount() {
   }
 }
 
-export function* fetchAccountAsync({ isSignUp = false }): SagaIterator {
+function* handleUploadTrack() {
```

Comment:
wow TIL, this feels like a big footgun lol

Reply:
I do _not_ love the cross saga listening patterns.